lib/filter/tests/Makefile.am

```diff
@@ -1,5 +1,6 @@
 lib_filter_tests_TESTS		 =              \
     lib/filter/tests/test_filters               \
+		lib/filter/tests/test_filters_facility      \
```

Comment:
Please unify the whitespaces. I do not have problem with either space or tabs. But with tabs chosen, one tab should be used.

Reply:
replaced with TAB